charts/redis-cluster/templates/registry-secret.yaml

```
@@ -0,0 +1,28 @@
+{{- if .Values.imageRegistrySecret.enabled }}
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: {{ include "redis-cluster.fullname" . }}-dockerconfig-regsec
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)